terms/folding_scheme.md

```diff
@@ -0,0 +1,11 @@
+Intuitively, a folding scheme for a relation $R$ is a protocol that reduces the task 
```

Comment:
Where is the definition of "relation"

Reply:
? It's in the chapter 12 of Proof, Arg and Zk. That is a part that you wrote.

Reply:
If you do not understand what the "relation" is, please check your work again.
I will separate that definition into a new file if missing the linked definition is what you mean.